### PR TITLE
Add first draft

### PR DESCRIPTION
**Pull request by viveknimkarde**
- Add first draft

### CODE REVIEW

File: docs/realtime-speaker-identification-api.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+| duration  | Number | Duration of chunk                       | optional |
+
+### Output Response Stream
+
+Output is the iterator streaming response objects with fowllowing parameters:
+
+| Parameter  | Type   | Description                           | Notes |
+| ---------- | ------ | ------------------------------------- | ----- |
+| speaker_id | String | speaker_id of the identified speaker. |
+| start      | Float  | Start of the audio segment.           |
```
Comment (sushant-hiray) [inline docs/realtime-speaker-identification-api.md]: Fix table

File: docs/realtime-speaker-identification-api.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+| sampleRate   | Number | Sample rate of the audio file.                   |
+| languageCode | String | Language spoken in the audio file.               | [default to &#39;en-US&#39;]                    |
+| speakerIds   | String | list of speakerIds to identify seperated by ','. |                                                 |
+
+### Segment Parameters
+
+| Parameter | Type   | Description                             | Notes    |
+| --------- | ------ | --------------------------------------- | -------- |
+| content   | String | base64 encoding of the audio segment.   |
+| offset    | Number | Segment offset from start of the audio. |
```
Comment (sushant-hiray) [inline docs/realtime-speaker-identification-api.md]: Fix table

File: docs/speaker-enrollment-api.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+  "message": "Success"
+}
+```
+
+### Body Parameters
+
+| Parameter    | Type   | Description                               | Notes                        |
+| ------------ | ------ | ----------------------------------------- | ---------------------------- |
+| encoding     | String | Encoding of audio file like MP3, WAV etc. |
+| sampleRate   | Number | Sample rate of the audio file.            |
```
Comment (sushant-hiray) [inline docs/speaker-enrollment-api.md]: Fix table

File: docs/speaker-identification-api.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+  }
+]
+```
+
+### Body Parameters
+
+| Parameter    | Type         | Description                               | Notes                        |
+| ------------ | ------------ | ----------------------------------------- | ---------------------------- |
+| encoding     | String       | Encoding of audio file like MP3, WAV etc. |
+| sampleRate   | Number       | Sample rate of the audio file.            |
```
Comment (sushant-hiray) [inline docs/speaker-identification-api.md]: Fix table

**Review by sushant-hiray — CHANGES_REQUESTED** [review]
Check the markdown tables

**Comment by viveknimkarde** [comment]
@sushant-hiray can you merge this PR

**Review by sushant-hiray — APPROVED** [review]

**Pull request merged**